test/cpp/ios/CronetTests/TestHelper.h

```diff
@@ -46,8 +46,8 @@ class DummyInterceptor : public grpc::experimental::Interceptor {
   static int GetNumTimesRun();
 
  private:
-  static std::atomic<int> num_times_run_;
-  static std::atomic<int> num_times_run_reverse_;
+  static std::atomic<int> num_times_run;
```

Comment:
This is still member variable of the class so if it was me I would have put `_` at the end... but maybe you removed it due to some style guide somewhere? Just a question; not really comment to the code.

Reply:
I wasn't sure about the guidelines for naming static members. I have reverted this change and restored the trailing `_`.